packages/toolbox/src/commands/chainlink/addAggregator.ts

```diff
@@ -57,32 +57,35 @@ export const builder = (): yargs.Argv<Options> =>
   });
 
 export const handler = async (args: Options): Promise<void> => {
-  let { input, output, aggregator } = args;
+  const { input, output, aggregator: aggregatorArg } = args;
   const { network, list } = args;
 
-  const currencyManager = await getCurrencyManager(list);
-
   EvmChains.assertChainSupported(network);
+  const currencyManager = await getCurrencyManager(list);
   const inputCcy = currencyManager.from(input, network) || currencyManager.from(input);
   const outputCcy = currencyManager.from(output, network) || currencyManager.from(output);
 
-  if (!input.startsWith('0x')) {
+  let inputAddress = input;
+  if (!inputAddress.startsWith('0x')) {
     assert(inputCcy, `input ${input} not found`);
-    input = inputCcy.hash;
+    inputAddress = inputCcy.hash;
   }
-  if (!output.startsWith('0x')) {
+
+  let outputAddress = input;
```

Comment:
Is this right? Shouldn't we set outputAddress to output?

```suggestion
  let outputAddress = output;
```